parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -49,6 +50,16 @@ pub mod pallet {
 	use frame_support::pallet_prelude::*;
 	use frame_system::pallet_prelude::*;
 
+	#[derive(scale_info::TypeInfo, codec::Encode, codec::Decode, codec::MaxEncodedLen)]
+	#[codec(mel_bound(T: Config))]
+	#[scale_info(skip_type_params(T))]
+	pub struct MaxFinalizedHeadersToKeep<T: Config>(PhantomData<T>);
+	impl<T: Config> Get<u32> for MaxFinalizedHeadersToKeep<T> {
+		fn get() -> u32 {
+			config::EPOCHS_PER_SYNC_COMMITTEE_PERIOD as u32 * 2
```

Comment:
1 `EPOCHS_PER_SYNC_COMMITTEE_PERIOD` should be enough and we keep 2 here for redundancy